src/sys/linux/errno.rs

```diff
@@ -0,0 +1,273 @@
+#![allow(dead_code)]
```

Comment:
nit: could the errno.py script be modified to add a header file that says "do not modify, automatically generated using errno.py"